docs/kratos/session-management/01_overview.mdx

```diff
@@ -27,113 +30,82 @@ For security reasons, you can't break the isolation between cookies and session
 | `active`           | When set to `true`, the Ory Session is active and can be used to authenticate requests.                                                                                                                                                                                                                                                                                                                                                                                                                          |
 | `expires_at`       | Defines the time when the Session expires. This value depends on the session lifespan configuration.                                                                                                                                                                                                                                                                                                                                                                                                             |
 | `authenticated_at` | Indicates the time of the most recent successful authentication. This value is updated when: <ul><li>The end-user authenticates with a second factor such as TOTP</li><li>The end-user refreshes their session using the [`/self-service/login/browser`](../../reference/api.mdx#operation/initializeSelfServiceLoginFlowForBrowsers) endpoint or [`/self-service/login/api`](../../reference/api.mdx#operation/initializeSelfServiceLoginFlowWithoutBrowser) endpoint and setting `refresh` to `true`</li></ul> |
 
-## Ory Session Cookie
+### Using Ory Session Cookie
 
 An Ory Session Cookie is issued when the user signs in through a web browser. To get the session payload, send a request to the
 `/sessions/whoami` endpoint:
 
+````mdx-code-block
+<Tabs groupId="sdk-console">
+<TabItem value="native" label="React">
+
 ```mdx-code-block
-import { getSdkUrl } from '@site/src/hooks'
-import CodeBlock from '@theme/CodeBlock'
+import reactWhoAmI from "!!raw-loader!@site/code-examples/sdk/typescript/src/selfservice/session/whoami-react.tsx"
+
+<CodeBlock language="ts" title="Checking current session">{reactWhoAmI}</CodeBlock>
+```
 
+</TabItem>
+<TabItem value="curl" label="cURL">
+
+
+```mdx-code-block
 <CodeBlock language="shell">{`curl '${getSdkUrl().url}/sessions/whoami' \\
   -H 'Accept: application/json' \\
   -H 'Cookie: ory_kratos_session=MTYzNDIyNzEzN3xEdi1CQkFFQ180SUFBUkFCRUFBQVJfLUNBQUVHYzNSeWFXNW5EQThBRFhObGMzTnBiMjVmZEc5clpXNEdjM1J5YVc1bkRDSUFJRTFDYWtvME5VNVlaVWxvYVZWeWJrUnZhSEF4YmxSV2VVRlhNMWwxVlVGenxXpsk2cL21Dclk3nCoXV41N6bFxvVJSt7CeICy_815Aw=='`}</CodeBlock>
 ```
 
-## Ory Session Token
+</TabItem>
+</Tabs>
+````
+
+### Using Ory Session Token
 
 An Ory Session Token is issued when the user authenticates through a client other than a web browser. To get the session payload,
```

Comment:
Also technically true, but a browser application can also call `login/api`. I think we should just be more verbose and say something along the lines of Native applications such as desktop and mobile applications, terminal based applications that do not run inside a browser should use tokens instead of cookies.